### PR TITLE
#329 Added axis labels and tooltip to graph.

### LINKED ISSUE #329: Add axis labels to graph
Remember that the labels need to update when the graph metric dropdown changes.

### PR DESCRIPTION
**Pull request by marijnvanderhorst**
#### What have I done
Fixes #329.
Adds axis labels to the graph. X-axis simply gets a static time label. Y-axis needs to update when the selected GraphMetric changes. Also formatted the tooltip when hovering over a point in the graph.

#### Testing
* Import data.
* Open graph.
* Check axis labels, also after switching GraphMetric.
* Also hover over data points to see the tooltip.

#### Addresses SRD section(s)
Sections about GraphMetric.

### CODE REVIEW

File: src/app/graph/graph-service/graph-metric.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    public static readonly VELOCITY = new GraphMetric('Velocity', 'mm/' + TimeService.UNIT);
+    public static readonly ACCELERATION = new GraphMetric('Acceleration', 'mm/' + TimeService.UNIT + '\u00B2'); // mm/{time}^2
+    public static readonly POSITION = new GraphMetric('Position', 'mm');
+    public static readonly DISTANCE_TO_NEXT = new GraphMetric('Distance to next', 'mm');
+    public static readonly DISTANCE_TO_PREVIOUS = new GraphMetric('Distance to previous', 'mm');
+
+    /**
+     * Name of this graph metric. May contain Unicode characters.
+     */
+    private readonly _name: string;
```
Comment (nolmoonen) [inline src/app/graph/graph-service/graph-metric.ts]: Justify deviations from naming scheme please.

File: src/app/graph/graph-service/graph-metric.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    /**
+     * Unit of this graph metric. May contain Unicode characters.
+     */
+    private readonly _unit: string;
```
Comment (nolmoonen) [inline src/app/graph/graph-service/graph-metric.ts]: Justify deviations from naming scheme please.

File: src/app/graph/graph-service/graph-metric.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
+/**
```
Comment (nolmoonen) [inline src/app/graph/graph-service/graph-metric.ts]: Comment should be below imports.

File: src/app/core/services/time.service.ts [hunk 4]
```diff
@@ -6,6 +6,16 @@ import { IllegalArgumentError } from '../error/illegal-argument-error';
 })
 export class TimeService {
 
+    /**
+     * The (abbreviated) unit of time that is used.
+     */
+    public static readonly UNIT: string = 's';
```
Comment (nolmoonen) [inline src/app/core/services/time.service.ts]: See general comment.

**Comment by nolmoonen** [comment]
Labels seem to work fine. 

While testing, encountered a somewhat related issue: #365 .

File: src/app/graph/graph-service/graph-metric.ts (hunk 3 above)
Comment (marijnvanderhorst) [reply src/app/graph/graph-service/graph-metric.ts]: Oepsie

File: src/app/graph/graph-service/graph-metric.ts (hunk 1 above)
Comment (marijnvanderhorst) [reply src/app/graph/graph-service/graph-metric.ts]: IMO we should justify deviation from the "TypeScript way". Added comment regardless.

File: src/app/graph/graph-service/graph-metric.ts (hunk 2 above)
Comment (marijnvanderhorst) [reply src/app/graph/graph-service/graph-metric.ts]: IMO we should justify deviation from the "TypeScript way". Added comment regardless.

File: src/app/core/services/time.service.ts (hunk 4 above)
Comment (marijnvanderhorst) [reply src/app/core/services/time.service.ts]: Let's agree to disagree. For consistency, I've also added the unit of distance as a const to `Path`.

**Pull request merged**